src/composite/control.jl

```diff
@@ -210,8 +210,10 @@ function Base.copy(ctrl::ControlBlock{N,BT,C,M}) where {BT,N,C,M}
 end
 
 function YaoBase.iscommute(x::ControlBlock{N}, y::ControlBlock{N}) where {N}
-    if x.locs == y.locs && x.ctrl_locs == y.ctrl_locs
```

Comment:
why this is removed?

Reply:
to commute two control blocks, ctrl locs are not nessesarily same.